ui/src/components/modal/owners-modal.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import {
+    AlertDialog,
+    AlertDialogHeader,
+    AlertDialogContent,
+    AlertDialogDescription,
+    AlertDialogTitle,
+    AlertDialogFooter
+} from '@/components/ui/alert-dialog';
+import { MouseEventHandler } from 'react';
+import { GroupingGroupMember } from '@/lib/types';
+import { Button } from '@/components/ui/button';
+
+const OwnersModal = ({
+    open,
+    close,
+    modalData
+}: {
+    open: boolean;
+    close: MouseEventHandler;
```

Comment:
The type for close seems incorrect, maybe close should be renamed to onClose and the type would be: () => void